packages/react-nexus/src/useNexus.tsx

```diff
@@ -10,6 +10,7 @@ const warningMessage =
 
 export default function useNexus<T = any, S = any>(
   apiCall: (nexus: NexusClient) => Promise<any>,
+  inputs?: any[],
```

Comment:
I'd like to sit down and think about that one. The idea of doing `useNexus(nexus => nexus.apiCall(args), someRandomInputs)` makes me uncomfortable. I feel like it is more down to your component implementation. But again, I don't have a concrete example in front of me.

Reply:
here's where I've used it.  https://github.com/BlueBrain/nexus-web/pull/272/files#diff-4af8cc18bb72eae3c19450fc7cd78c25R20

Otherwise this useNexus will be minimally useful because it won't update it's values, even when the input project label has changed, for example